python_modules/libraries/dagster-airbyte/dagster_airbyte/resources.py

```diff
@@ -791,3 +794,114 @@ def airbyte_cloud_resource(context) -> AirbyteCloudResource:
 
     """
     return AirbyteCloudResource.from_resource_context(context)
+
+
+# ------------------
+# Reworked resources
+# ------------------
+
+
+@whitelist_for_serdes
+@record
+class AirbyteConnection:
+    """Represents an Airbyte connection, based on data as returned from the API."""
+
+    @classmethod
+    def from_connection_details(
+        cls,
+        connection_details: Mapping[str, Any],
+    ) -> "AirbyteConnection":
+        raise NotImplementedError()
+
+
+@whitelist_for_serdes
+@record
+class AirbyteDestination:
+    """Represents an Airbyte destination, based on data as returned from the API."""
+
+    @classmethod
+    def from_destination_details(
+        cls,
+        destination_details: Mapping[str, Any],
+    ) -> "AirbyteDestination":
+        raise NotImplementedError()
+
+
+@record
+class AirbyteWorkspaceData:
```

Comment:
The workspace data is not specific to Airbyte Cloud, which is why this object is named `AirbyteWorkspaceData`.

Reply:
feels worth mentioning that in the docstring, that it applies to both airbyte OSS and airbyte cloud potentially? Or at least a comment